CHANGELOG.md

```diff
@@ -1,11 +1,174 @@
+
+Finally, as is the case with session recording, please report any bugs in the functionality on [GitHub](https://github.com/PostHog/posthog/issues).
+
+
+- [Multiple Projects](https://github.com/PostHog/posthog/pull/1562)
+
+![Multiple Projects Screenshot](https://posthog.com/static/821f1e938621ad7d37e1ce0e9a3704a9/2cefc/org-project.png)
+
+You asked and we delivered!
+
+As per feedback from many in our community, PostHog now offers support for managing multiple projects under one "umbrella" organization. 
+
+This allows you to segregate concerns, such as keeping tracking for your dev and prod environments separately, as well as track multiple domains and apps without mixing data.  
+
+In addition, we also enhanced our invite and permissioning system as a by-product of this feature. 
+
+As this is an Enterprise Edition feature, please contact us at _sales@posthog.com_ if you are interested in using it.
+
+- [Dashboard Templates](https://github.com/PostHog/posthog/pull/1942)
+
+![Dashboard Templates Screenshot](https://posthog.com/static/1430069845eb4f0a34a7d4afc9b9fa30/2cefc/dashboard-template.png)
+
+In order to make it easier to create valuable dashboards to keep track of your business metrics, PostHog now offers the option to create new dashboards based on a template. We will be expanding the power of dashboard templates, but, as of right now, you can already create a dashboard using our web app dashboard template, which provides you with a good starting point for determining and tracking relevant metrics.
+
+- [Setup Improvements](https://github.com/PostHog/posthog/pull/1990)
+
+![Google Login Screenshot](https://posthog.com/static/14e1269485fcf90f90e761b1accbeb34/2cefc/google-login.png)
+
+In addition to GitHub and GitLab authentication, PostHog now supports signup and login with Google accounts! 
+
+We also improved our setup process by better structuring our settings pages, allowing you to [change your project's token](https://github.com/PostHog/posthog/pull/2015), and [enhancing the UX for empty states on dashboards](https://github.com/PostHog/posthog/pull/2068).
+
+- [Documentation Level Up](https://github.com/PostHog/posthog.com)
+
+![Docs Screenshot](https://posthog.com/static/5b1046b5a6615c1cd91af2519b8d603d/2cefc/docs.png)
+
+We have been working hard to improve our product documentation and had a few big upgrades recently:
+
+- Our Docs now have a Dark Mode option
+- You can search our entire documentation without ever using your mouse
+- We are actively releasing new tutorials on how to use PostHog to track key metrics and improve your product
+- Our Docs pages now load faster
+- New screenshots have been added throughout the Docs, as well as functionality walkthrough videos
+
+...and a lot more!
+
+If you have any suggestions for new tutorials or improvements to our documentation, [do not hesitate to let us know!](https://github.com/PostHog/posthog.com/issues)
+
+- [User Interviews](calendly.com/posthog-feedback)
```

Comment:
This link doesn't work (needs `https://`) and I think is redundant, since it's also below.